pandas/core/generic.py

```diff
@@ -7576,10 +7598,10 @@ def resample(self, rule, how=None, axis=0, fill_method=None, closed=None,
         specify on level the resampling needs to take place.
 
         >>> time = pd.date_range('1/1/2000', periods=5, freq='T')
-        >>> df2 = pd.DataFrame(data=10*[range(4)],
-                               columns=['a', 'b', 'c', 'd'],
-                               index=pd.MultiIndex.from_product([time, [1, 2]])
-                               )
+        >>> df2 = pd.DataFrame(data=10 * [range(4)],
+        ...                    columns=['a', 'b', 'c', 'd'],
+        ...                    index=pd.MultiIndex.from_product([time, [1, 2]])
+        ...                    )
         >>> df2.resample('3T', level=0).sum()
```

Comment:
same comments as in the previous example.

Reply:
Done, also with a different, more realistic approach. Please let me know your comments. I also struggled a bit with the formatting of long lines, opted for styling lines 7627-7632 using [black](https://github.com/ambv/black) (also compatible with PEP8) but let me know it breaks consistency and should be reverted to the old style. Pushing now.